src/ucs/datastruct/bitmap.h

```diff
@@ -90,7 +90,7 @@ _ucs_bitmap_word_index(size_t bitmap_words, size_t bit_index)
  * @param _bitmap    Take the word from this bitmap
  * @param _bit_index Index of the bit for fetching the word
  *
- * @return The word which containt
+ * @return The word which contains
```

Comment:
This sentence is weird (it was weird from the beginning). Maybe improve it?

Reply:
ok, updated